### PR TITLE
gtklock: update to 1.3.1

### PR DESCRIPTION
**Pull request by jovanlanik**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Comment by jovanlanik** [comment]
Hmmm, seems I did something wrong.

**Comment by paper42** [comment]
> Hmmm, seems I did something wrong.

That's not your mistake, wait until all the builders finish building and then it should work.

**Comment by jovanlanik** [comment]
You were right 😅

File: srcpkgs/gtklock/template [hunk 1]
```diff
@@ -11,7 +11,7 @@ maintainer="Jovan Lanik <jox969@gmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/jovanlanik/gtklock"
 distfiles="${homepage}/archive/refs/tags/v${version}.tar.gz"
```
Comment (classabbyamp) [inline srcpkgs/gtklock/template:13]: expand $homepage here

**Pull request merged**